siddhar_iyal/description.en.utf8

```diff
@@ -0,0 +1,22 @@
+<h1>Vinniyalum Valviyalum's Siddhar Iyal Sky Chart</h1>
```

Comment:
the ".en." in the filename indicates this to be the English version. It is a copy of the Czech version above. 

The English text is the most important description. It is the master from which translators will hopefully create translations to other languages.